packages/semantic-ui/package.json

```diff
@@ -115,6 +113,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "nanoid": "^3.1.23",
```

Comment:
We should keep nanoid as a peerDependency for the semantic-ui package (just like what we had originally for nanoid).